### DISCUSSION TITLE
"Invalid authentication credentials" in self-hosted environment

### DISCUSSION THREAD

**Discussion by xpointsolucoes** [supabase/supabase#21846] [Questions]
I'm folowing [this](https://supabase.com/docs/guides/self-hosting/docker) documentation, the unique change i've made was in the .env file to update the `JWT_SECRET`, `ANON_KEY ` and `SERVICE_ROLE_KEY `. 
i've tested this on a react native application and in Postman, on both of then i receive the same message. Thats my latest request:
```
curl --location 'http://127.0.0.1:8000/rest/v1' \
--header 'apiKey: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...'
```
i've made the request for this url too `http://127.0.0.1:8000/rest/v1/admin/users`, same response
i tried to re-generate the keys but no success at all. Is there any other change i need to do for make this work?

**Comment by ManuelAngel99** [comment]
Same issue here, seems like the online `ANON_KEY`  and `SERVICE_ROLE_KYE` generator is not working properly at the moment

**Reply by BadMask121** [reply]
same here got the same errors i havent been able to use it with next-auth
